mantis-common/src/main/java/io/mantisrx/common/metrics/measurement/Measurements.java

```diff
@@ -74,6 +95,25 @@ public String toString() {
                 ", tags=" + tags +
                 ", counters=" + counters +
                 ", gauges=" + gauges +
+                ", micrometers=" + micrometers +
                 '}';
     }
+
+    public static class MicrometerMeasurement {
```

Comment:
this is a good change, umm, minor thing if I may suggest is to create it as a separate class similar to `GaugeMeasurement`